x-pack/packages/kbn-data-forge/src/data_sources/fake_hosts/ecs/fields/custom/system.yml

```diff
@@ -20,6 +20,18 @@
       level: custom
       type: float
       description: "Percentage of CPU usage by system processes"
+    - name: load.1
+      level: custom
+      type: float
+      description: "Load 1m by system processes"
+    - name: memory.actual.used.pct
+      level: custom
+      type: float
+      description: "Percentage of actual memory by system processes"
+    - name: filesystem.used.pct
+      level: custom
+      type: float
+      description: "Percentage of filesytem used by system processes"
```

Comment:
These were missing from the mappings and causing errors when the first values was a `long` but the subsequent values were `floats`